specs/graph-protocol-hybrid-network/messages/README.md

```diff
@@ -97,6 +98,9 @@ Sent if the result of calling the read operation is larger than the `maxBytes` p
 | Field Name | Field Type | Description |
 | ---------- | ---------- | ----------- |
 | status | String | The constant "MAX_BYTES_EXCEEDED" |
+| attestation | Object | An Attestation, where the `requestCID` is the CID of the object containing the above field. |
+
```

Comment:
You probably need to allow a `Request Declined` response in the channel to accompany the out-of-channel response below. This makes it clear that the request is not outstanding, freeing up funds to be used for future requests.

Reply:
Couldn't the user just send a follow-on request with the same nonce/turn value?